test/utils/test.utils.ts

```diff
@@ -0,0 +1,6 @@
+export function wordCount(wordToCount: string, inText: string): number {
+  const regex = new RegExp(`\\b${wordToCount}\\b`, 'gi');
+  const matches = inText.match(regex);
+  const wordCount = matches ? matches.length : 0;
+  return wordCount;
+}
```

Comment:
I wanted to be able to count occurrences of a word in text in the tests, so created this test.utils.ts file for this and other 'helper' test functions